spec.html

```diff
@@ -119,12 +119,12 @@ <h1>AggregateError ( _errors_, _message_ )</h1>
         <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
         <emu-alg>
           1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
-          1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]], [[AggregateErrors]] »).
+          1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
           1. Let _errorsList_ be ? IterableToList(_errors_).
-          1. Set _O_.[[AggregateErrors]] to _errorsList_.
           1. If _message_ is not _undefined_, then
             1. Let msg be ? ToString(_message_).
             1. Perform ! CreateMethodProperty(_O_, `"message"`, _msg_).
+          1. Perform ! CreateMethodPropertyOrThrow(_O_, `"errors"`, ! CreateArrayFromList(_errorsList_)).
```

Comment:
I continue to think `CreateMethodProperty` is weird to use here, since this is not a method. The other error constructors [explicitly make an appropriate property descriptor](https://tc39.es/ecma262/#sec-nativeerror).

Reply:
alrighty

Reply:
Good call. The initial version of the proposal used `CreateMethodProperty` as well. I agree avoiding it here seems better. Thanks for making the change!